streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java

```diff
@@ -462,7 +462,7 @@ public void process(final Integer key, final byte[] value) {
         final KafkaStreams streams = createKafkaStreamsWithExceptionHandler(builder, props);
         runGenericBenchmark(streams, "Streams Stateful Performance [records/latency/rec-sec/MB-sec joined]: ", latch);
     }
-
+    @SuppressWarnings("deprecation")
```

Comment:
We should migrate away from deprecated APIs in our internal code.

Reply:
```suggestion
    removed @SuppressWarnings("deprecation")
    and used the non deprecated one store.put(key, value, context().timestamp());
```